### PR TITLE
Adding link to Jenny Bryans Happy Git and GitHub for the useR

### PR DESCRIPTION
**Pull request by slackline**
I've found this an excellent resource and thought it would complement the existing set.

### CODE REVIEW

**Review by Lextuga007 — CHANGES_REQUESTED** [review]
Great resource and perfect addition to the book. Could you put the sentence on a new line though - it's a style suggested from Turing Way so you might not be familiar with it. It helps with any future changes so that only change one line. In this case the sentence before won't be affected for example.

https://nhsrway.nhsrcommunity.com/style-guides.html#write-each-sentence-in-a-new-line-line-breaks

File: technical-git.qmd [hunk 1]
```diff
@@ -8,7 +8,7 @@ NHS RAP Community of Practice have a [Git Quick Start Guide](https://nhsdigital.
 
 ## Set up using R
 
-Following course materials developed by [R Forwards](https://forwards.github.io/workshops/package-dev-modules/slides/02-setting-up-system/setting-up-system.html#1) or [NHS-R Community Introduction to Git and GitHub using R](https://intro-git-github.nhsrcommunity.com/) which is also based on R Forwards slides.
+Following course materials developed by [R Forwards](https://forwards.github.io/workshops/package-dev-modules/slides/02-setting-up-system/setting-up-system.html#1) or [NHS-R Community Introduction to Git and GitHub using R](https://intro-git-github.nhsrcommunity.com/) which is also based on R Forwards slides. Another excellent resource is Jenny Bryan's [Happy Git and GitHub for the useR](https://happygitwithr.com/).
```
Comment (slackline) [inline technical-git.qmd:11]:
```suggestion
Following course materials developed by [R Forwards](https://forwards.github.io/workshops/package-dev-modules/slides/02-setting-up-system/setting-up-system.html#1) or [NHS-R Community Introduction to Git and GitHub using R](https://intro-git-github.nhsrcommunity.com/) which is also based on R Forwards slides.
Another excellent resource is Jenny Bryan's [Happy Git and GitHub for the useR](https://happygitwithr.com/).
```

**Comment by slackline** [comment]
Not a problem done.

Hadn't  clocked the contributing guidelines nor the come across the Turing Ways linting rules before (am aware of the Turing Way though, great resource). I normally apply [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) to my Markdown in projects/blogs/Quarto/etc. typically via a [pre-commit](https://pre-commit.com/) hook which doesn't start sentences on new lines but does hard-wrap long lines to achieve the same effect.

**Review by Lextuga007 — APPROVED** [review]
Thanks! Much appreciated.

**Comment by Lextuga007** [comment]
Good point on the pre-commit hooks. It's not something I've set up myself and would welcome anything technical like that for the book 😃

**Comment by slackline** [comment]
I'm a big fan of `pre-commit` and have [blogged](https://blog.nshephard.dev/#category=pre-commit) about it a few times and on the back of that had was asked to put together a talk for Leeds Research Software Engineering (I work as a Research Software Engineer at University of Sheffield, this is my personal Github account I also go by @ns-rse for work).

I'll see if I can find some time to write something and also setup a config for this repository, pretty busy with work at the moment and as the weather improves getting outdoors but will try and get something done.

**Pull request merged**